types/three/three-core.d.ts

```diff
@@ -5389,17 +5389,72 @@ export class Skeleton {
     clone(): this;
 }
 
+/**
+ * A mesh that has a Skeleton with bones that can then be used to animate the vertices of the geometry. The material must support skinning and have skinning enabled.
+ * @see MeshStandardMaterial.skinning
+ * @source https://github.com/mrdoob/three.js/blob/master/src/objects/SkinnedMesh.js
+ */
 export class SkinnedMesh extends Mesh {
-    constructor(geometry?: Geometry | BufferGeometry, material?: Material | Material[], useVertexTexture?: boolean);
 
+    /**
+     * @param  {BufferGeometry} geometry? an instance of {@link BufferGeometry}.
```

Comment:
Given that you are making a change that reflects `three.js@r99`, you should update the version in the first line of `index.d.ts` to 0.99.

Reply:
Thanks for reviewing, I don´t know if all the changes already made in that library reflects r99. Maybe it is a task for a reviewer to do that.